pkg/client/client.go

```diff
@@ -454,6 +454,11 @@ func (c *APIClient) decode(v interface{}, b []byte, contentType string) (err err
 		_, err = (*f).Seek(0, io.SeekStart)
 		return
 	}
+
+	// TODO: Hack for a bug where we sometimes get contentType XML back even though the content is actually JSON
+	// the content is always JSON in this version of the API client
+	contentType = "application/json"
```

Comment:
Before we force this content type change, I would like to see a log message indicating what the contentType is and that we are forcing a change to application/json. With that, I am good with the pull request.